From 08a78dd906c9645a762883bc1ccf90cb2900e599 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sat, 12 Jul 2025 02:21:51 -0400
Subject: Add /opt

---
 libtool | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/libtool b/libtool
index db9a983..72b261d 100644
--- a/libtool
+++ b/libtool
@@ -300,7 +300,7 @@ finish_eval=""
 hardcode_into_libs=no
 
 # Compile-time system search path for libraries.
-sys_lib_search_path_spec="/usr/lib/gcc/x86_64-pc-cygwin/15.1.0 /usr/x86_64-pc-cygwin/lib /usr/lib  /opt/usr/lib/w32api"
+sys_lib_search_path_spec="/opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1 /opt/usr/x86_64-pc-cygwin/lib /opt/usr/lib /usr/lib  /opt/usr/lib/w32api"
 
 # Detected run-time system search path for libraries.
 sys_lib_dlsearch_path_spec="/opt/usr/lib /usr/lib /lib"
@@ -12263,18 +12263,18 @@ file_list_spec="@"
 hardcode_action=immediate
 
 # The directories searched by this compiler when creating a shared library.
-compiler_lib_search_dirs="/usr/lib/gcc/x86_64-pc-cygwin/15.1.0 /usr/lib/gcc/x86_64-pc-cygwin/15.1.0/../../../../x86_64-pc-cygwin/lib /usr/lib/gcc/x86_64-pc-cygwin/15.1.0/../../.. /usr/lib"
+compiler_lib_search_dirs="/opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1 /opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1/../../../../x86_64-pc-cygwin/lib /opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1/../../.. /opt/usr/lib /usr/lib"
 
 # Dependencies to place before and after the objects being linked to
 # create a shared library.
-predep_objects="/usr/lib/gcc/x86_64-pc-cygwin/15.1.0/crtbeginS.o"
-postdep_objects="/usr/lib/gcc/x86_64-pc-cygwin/15.1.0/crtend.o"
+predep_objects="/opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1/crtbeginS.o"
+postdep_objects="/opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1/crtend.o"
 predeps=""
 postdeps="-lstdc++ -lgcc_s -lgcc -lmsys-2.0 -ladvapi32 -lshell32 -luser32 -lkernel32 -lgcc_s -lgcc"
 
 # The library search path used internally by the compiler when linking
 # a shared library.
-compiler_lib_search_path="-L/usr/lib/gcc/x86_64-pc-cygwin/15.1.0 -L/usr/lib/gcc/x86_64-pc-cygwin/15.1.0/../../../../x86_64-pc-cygwin/lib -L/usr/lib/gcc/x86_64-pc-cygwin/15.1.0/../../.. -L/usr/lib"
+compiler_lib_search_path="-L/opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1 -L/opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1/../../../../x86_64-pc-cygwin/lib -L/opt/usr/lib/gcc/x86_64-pc-cygwin/13.4.1/../../.. -L/opt/usr/lib -L/usr/lib"
 
 # ### END LIBTOOL TAG CONFIG: CXX
 
-- 
