From 58eadcdbe3ccad825742737210494be6f5df594b Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sat, 12 Jul 2025 02:46:03 -0400
Subject: LD="/opt/usr/x86_64-pc-cygwin/bin/ld.exe"

---
 libtool | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/libtool b/libtool
index 72b261d..5aea6cb 100644
--- a/libtool
+++ b/libtool
@@ -323,7 +323,7 @@ striplib="strip --strip-unneeded"
 
 
 # The linker used to build libraries.
-LD="/usr/x86_64-pc-cygwin/bin/ld.exe"
+LD="/opt/usr/x86_64-pc-cygwin/bin/ld.exe"
 
 # How to create reloadable object files.
 reload_flag=" -r"
@@ -12126,7 +12126,7 @@ build_old_libs=`case $build_libtool_libs in yes) echo no;; *) echo yes;; esac`
 # ### BEGIN LIBTOOL TAG CONFIG: CXX
 
 # The linker used to build libraries.
-LD="/usr/x86_64-pc-cygwin/bin/ld.exe"
+LD="/opt/usr/x86_64-pc-cygwin/bin/ld.exe"
 
 # How to create reloadable object files.
 reload_flag=" -r"
@@ -12579,7 +12579,7 @@ compiler_lib_search_path=""
 # ### BEGIN LIBTOOL TAG CONFIG: GO
 
 # The linker used to build libraries.
-LD="/usr/x86_64-pc-cygwin/bin/ld.exe"
+LD="/opt/usr/x86_64-pc-cygwin/bin/ld.exe"
 
 # How to create reloadable object files.
 reload_flag=" -r"
@@ -12728,7 +12728,7 @@ compiler_lib_search_path=""
 # ### BEGIN LIBTOOL TAG CONFIG: GCJ
 
 # The linker used to build libraries.
-LD="/usr/x86_64-pc-cygwin/bin/ld.exe"
+LD="/opt/usr/x86_64-pc-cygwin/bin/ld.exe"
 
 # How to create reloadable object files.
 reload_flag=" -r"
-- 
